packages/verification-vue-component/src/components/Verification/VerificationDropBox.vue

```diff
@@ -1,53 +1,47 @@
 <template>
-    <div class="verification-dropbox" :class="{dragover:draggingOver}"
-         @dragover.prevent="draggingOver = true"
-         @dragleave="draggingOver = false"
-         @drop="handleDrop">
+    <div class="verification-dropbox" @drop="handleDrop">
```

Comment:
Do you need to handle the drop event here?

Reply:
You are right. I had here a wrong thought, changed it acc. Drop is handled now on CertifactionVerification level